03.bowling/bowling.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+def main
+  scoreboard = ARGV[0]
+  frames = split_by_frames(scoreboard)
+  final_score = calculate_score(frames)
+  output_result(final_score)
+end
+
+def spare?(frame)
+  true if frame[0] + frame[1] == 10
+end
+
+def strike?(frame)
+  true if frame[0] == 10
```

Comment:
```suggestion
  frame[0] == 10
```

`==`の戻り値がtrue/falseになるのでこれだけでOKです